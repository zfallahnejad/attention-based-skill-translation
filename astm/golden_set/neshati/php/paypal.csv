367220,paypal
1783977,paypal
3253090,paypal
250259,paypal
2402976,paypal
557846,paypal
735556,paypal
2026139,paypal
