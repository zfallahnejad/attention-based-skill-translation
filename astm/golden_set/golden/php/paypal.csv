367220,73,61,0.5447761194029851
1783977,33,46,0.4177215189873418
3253090,22,18,0.55
250259,10,3,0.7692307692307693
557846,8,3,0.7272727272727273
2402976,8,7,0.5333333333333333
735556,7,9,0.4375
2026139,7,8,0.4666666666666667
